BooruSharp.Others/Pixiv.cs

```diff
@@ -25,33 +25,40 @@ public class Pixiv : ABooru
         public async Task LoginAsync(string username, string password)
         {
             var request = new HttpRequestMessage(HttpMethod.Post, "https://oauth.secure.pixiv.net/auth/token");
+
             string time = DateTime.UtcNow.ToString("yyyy-MM-ddTHH:mm:ss+00:00");
             request.Headers.Add("X-Client-Time", time);
+
             using (var md5 = MD5.Create())
             {
-                var bytes = md5.ComputeHash(Encoding.UTF8.GetBytes(time + _hashSecret));
-                StringBuilder builder = new StringBuilder();
-                for (int i = 0; i < bytes.Length; i++)
-                    builder.Append(bytes[i].ToString("x2"));
-                request.Headers.Add("X-Client-Hash", builder.ToString());
+                var hashBytes = md5.ComputeHash(Encoding.UTF8.GetBytes(time + _hashSecret));
+                var hashString = BitConverter.ToString(hashBytes).Replace("-", null).ToLower();
+
+                request.Headers.Add("X-Client-Hash", hashString);
```

Comment:
Could you please make sure these lines aren't the reason the Pixiv unit tests are failing?

Reply:
Hex string output is the same. See https://dotnetfiddle.net/17PSj2